src/Tgstation.Server.Host/Components/Events/EventType.cs

```diff
@@ -137,5 +137,11 @@ public enum EventType
 		/// </summary>
 		[EventScript("WorldPrime")]
 		WorldPrime,
+
+		/// <summary>
+		/// After the watchdog has launched successfully. Parameters: PID of DreamDaemon
+		/// </summary>
+		[EventScript("WatchdogSuccessfulLaunch")]
+		WatchdogSuccessfulLaunch,
```

Comment:
Call it DreamDaemonLaunch, at the time you call the event, you never checked to see if the launch was successful